internal/graph/graph.go

```diff
@@ -115,6 +115,7 @@ type RelationshipIngress struct {
 }
 
 func (r RelationshipIngress) String() string {
+	// TODO log the condition
```

Comment:
I don't know that the ingress itself should ever have a logger associated with it. If you want to log the condition, that should be the callers responsibility, and the caller would presumably have a logger at that point..

Can we remove this comment?

Reply:
oh i meant `print` the condition, not log

Reply:
Why would we print the condition in the `String()` method of the ingress? Shouldn't that be the callers responsibility to print the `ingress.String()`?

Can't we just remove this comment? What follow up todo is there to be done here?

Reply:
> Why would we print the condition in the String() method of the ingress?

Because the `condition` is a field of the `RelationshipIngress`, so why does it NOT print it is my question 😅

Reply:
Oh, gosh. We're not communicating well here. We're not talking about "printing" anything. We're talking about adding the `condition` field to the returned string. Returned vs printed is the miscomm here 😄

Reply:
Yes :D hahaha